src/controllers/post-test/post-test.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { type Request, type Response } from 'express'
```

Comment:
I have to import types from express. If I dont't do this, typescript dont recognise this types properly.
Or you mean to import like this import {  Request, Response } from 'express'?
But it's not suitable for my case, because linter is checking: "All imports in the declaration are only used as types. Use `import type`.eslint[@typescript-eslint/consistent-type-imports](https://typescript-eslint.io/rules/consistent-type-imports)"